content/courses/mobile/intro-to-solana-mobile.md

```diff
@@ -935,47 +945,45 @@ to CSS.
 In `screens/MainScreen.tsx` paste the following:
 
 ```tsx
+import React from "react";
 import { StatusBar, StyleSheet, View } from "react-native";
 import { CounterView } from "../components/CounterView";
 import { CounterButton } from "../components/CounterButton";
-import React from "react";
 
-const mainScreenStyles = StyleSheet.create({
+export function MainScreen() {
+  return (
+    <View style={styles.container}>
+      <StatusBar barStyle="light-content" backgroundColor="darkblue" />
+      <View style={[styles.container, styles.counterContainer]}>
+        <CounterView />
+      </View>
+      <View style={styles.incrementButtonContainer}>
+        <CounterButton />
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
   container: {
     height: "100%",
     width: "100%",
     backgroundColor: "lightgray",
   },
-
-  incrementButtonContainer: { position: "absolute", right: "5%", bottom: "3%" },
+  incrementButtonContainer: {
+    position: "absolute",
+    right: "5%",
+    bottom: "3%",
+  },
   counterContainer: {
     alignContent: "center",
     alignItems: "center",
     justifyContent: "center",
   },
 });
-
-export function MainScreen() {
-  return (
-    <View style={mainScreenStyles.container}>
-      <StatusBar barStyle="light-content" backgroundColor="darkblue" />
-      <View
-        style={{
-          ...mainScreenStyles.container,
-          ...mainScreenStyles.counterContainer,
-        }}
-      >
-        <CounterView />
-      </View>
-      <View style={mainScreenStyles.incrementButtonContainer}>
-        <CounterButton />
-      </View>
-    </View>
-  );
-}
 ```
 
-#### 9. Create CounterView.tsx
+### 9. `Create CounterView.tsx`
```

Comment:
No backticks or special characters in titles, see https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#heading-styles